ral-facilities/operationsgateway
# DSEGOG-174 saving plots to redux store

## Description
This allows for the user to save the plot they have open in a window to the Redux store. This lasts while the main window is not closed or refreshed, since sessions have yet to be setup.

The Save As button has been removed as I didn't deem it useful. Instead, the Save button saves the plot under the title it currently has. If a plot is renamed and saved, a new saved plot is stored in Redux, with the old copy remaining under the old name.

When the user saves a plot and closes it, it can be reopened with its state restored from Redux.

## Testing instructions
Try saving a plot and reopening it with the Edit button. All state should be restored.


## Agile board tracking
Closes DSEGOG-174
